src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/SymbolFactory.vb

```diff
@@ -49,6 +49,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
             Return False
         End Function
 
+        Friend Overrides Function IsConstModifierType(moduleSymbol As PEModuleSymbol, type As TypeSymbol) As Boolean
+            ' VB doesn't deal with ref-readonly parameters or return-types.
+            Return False
```

Comment:
If that's the case then we should consider renaming both of these methods.  The name simply does not match up to the actual implementation.  Consider this code

``` c#
var isConst = IsConstModifierType(type);
```

99% of developers who look at that line are going to interpret this as determining if `type` is the const modifier. In reality it's detecting if a) it's the const modifier and b) the present language supports it.  The 1% who will understand the behavior are those who realize the name of the method is wrong.